xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -923,16 +1674,16 @@ bool CGUIFontTTF::CacheCharacter(FT_UInt glyphIndex, uint32_t style, Character*
   }
 
   // set the character in our table
-  ch->m_glyphAndStyle = (style << 16) | glyphIndex;
+  ch->m_glyphAndStyle = (static_cast<decltype(ch->m_glyphAndStyle)>(style) << 32) | glyphIndex;
```

Comment:
Seems more clear:

`ch->m_glyphAndStyle = (static_cast<glyph_and_style_t>(style) << 32) | glyphIndex;`

Reply:
Have changed, was gone down where I changed the other. By other place was come a compile warning with it.